docs/fidesops/docs/guides/saas_config.md

```diff
@@ -493,7 +493,7 @@ update:
           direction: from
 ```
 
-Fidesops will replace the `<masked_object_fields>` placeholder with the result of the policy-driven masking service, for example `{'company': None, 'email': None}`.
+Fidesops will replace the `<masked_object_fields>` placeholder with the result of the policy-driven masking service, for example `'company': None, 'email': None`. Note that neither enclosing curly brackets (`{` `}`) nor a trailing comma (`,`) are included as part of the replacement string generated by Fidesops,
```

Comment:
Just a copy edit to replace the comma at the end of this line with a full stop `.` - otherwise this is good to go!

Reply:
whoops, good catch - thanks :)